var/spack/repos/builtin/packages/py-click-repl/package.py

```diff
@@ -11,7 +11,7 @@ class PyClickRepl(PythonPackage):
     homepage = "https://github.com/click-contrib/click-repl"
     pypi = "click-repl/click-repl-0.1.6.tar.gz"
 
-    version("0.2.0", sha256="b0cac32a625c24cd1414cc323e314a79278e2310e41596a6e27997e1c9f99e72")
+    version("0.2.0", sha256="cd12f68d745bf6151210790540b4cb064c7b13e571bc64b6957d98d120dacfd8")
```

Comment:
Looks like this old checksum must have come from the GitHub tarball instead of the PyPI tarball, good catch